spec/services/default_shipping_category_spec.rb

```diff
@@ -4,18 +4,18 @@
   describe '.create!' do
     it "names the location 'Default'" do
       shipping_category = described_class.create!
-      expect(shipping_category.name).to eq('Default')
+      expect(shipping_category.name).to eq 'Default'
     end
   end
 
   describe 'find_or_create' do
     context 'when a Default category already exists' do
-      let!(:category) do
+      let! :category do
```

Comment:
you need to keep the parenthesis here to keep consistency with all other specs.
in OFN code, not using parenthesis is almost only used in specs after eq